.github/workflows/main.yml

```diff
@@ -43,6 +43,10 @@ jobs:
             -Dmaven.javadoc.skip=true \
             -Djacoco.skip=true
 
+      - name: Publish Test Report
+        if: success() || failure()
```

Comment:
just wondering if this is required since we do it for both cases.
(although the example from scacap uses it like that).

Reply:
Workflows by default stop on a failure. This is needed to ensure the task gets executed if there are some.